### PR TITLE
AddToFavorite/AddToPlaylist HW-17

### PR DESCRIPTION
**Pull request by DanilKrivulin**

### CODE REVIEW

File: src/test/java/addtoPlaylist.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+
+import java.security.Key;
+import java.time.Duration;
+
+public class addtoPlaylist extends BaseTest {
```
Comment (Berichicko) [inline src/test/java/addtoPlaylist.java:13]: **According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example` public class PlaylistTest`

File: src/test/java/addtoPlaylist.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    }
+
+
+
+
+
+
+
+
```
Comment (Berichicko) [inline src/test/java/addtoPlaylist.java:49]: **Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

**Comment by Berichicko** [comment]
Hello 
Homework 17 is cheeked!
Fantastic job! Magnificent! 
💖Best wishes, the team HW (Alex,  aliaksandr@testpro.io)!💖

image

File: src/test/java/addtoPlaylist.java (hunk 1 above)
Comment (DanilKrivulin) [reply src/test/java/addtoPlaylist.java:13]: Ok, Thank you

Comment (Berichicko) [reply src/test/java/addtoPlaylist.java:13]: > Ok, Thank you

I'm glad you found it helpful

**Pull request closed without merging**